product_setup_page/src/CyencesAlertSetup.jsx

```diff
@@ -33,7 +71,10 @@ export default function CyencesAlertSetup() {
             <div style={{ marginLeft: '250px' }}>
                 {ComplianceTeamConfigurationMacros.map((macroItem) => <MacroSetup key={macroItem.name} macroName={macroItem.name} description={macroItem.description} />)}
             </div>
-
+            
+            <div style={{ marginLeft: '30px' }}>
+                <b>NOTE:</b> Would you like separate digest alert for the common recipients of SOC and Compliance?  <Switch inline key={SeparateDigestMacro} value={SeparateDigestMacro} selected={isEnabled} appearance="toggle" onClick={updateMacro}></Switch>
```

Comment:
**Mention note that** disabling this option would mean common recipient between SOC and Compliance will receive duplicate digest alerts.